google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/NamedResourceInspectionTest.java

```diff
@@ -58,7 +59,7 @@ public void testMethodsWithUnnamedResourcesEmptyString() {
 
   /**
    * Tests that the NamedResourceInspection's quick fix flagged with
-   * {@link NamedResourceInspection.Error.DUPLICATE_PARAMETER} for an @Named annotation updates
+   * {@link NamedResourceError#DUPLICATE_PARAMETER} for an @Named annotation updates
```

Comment:
nit: Do you mind running the formatter on this javadoc (and the ones below) while you're here?